app/presenters/document/web_resource_presenter.rb

```diff
@@ -165,17 +165,40 @@ def downloadable?
       end
     end
 
+    def for_edm_object?
+      @record_presenter.edm_object == url
+    end
+
+    def for_edm_is_shown_by?
+      url == @record_presenter.edm_is_shown_by
+    end
+
+    def displayable?
```

Comment:
Cyclomatic complexity for displayable? is too high. [11/6]<br>Assignment Branch Condition size for displayable? is too high. [19.72/15]<br>Perceived complexity for displayable? is too high. [11/7]